docs/shared_parsers_catalog/mpirun_ver.rst

```diff
@@ -0,0 +1,3 @@
+.. automodule:: insights.parsers.mpirun
```

Comment:
The filename should be the same as the parser filename i.e `mpirun.rst`

Reply:
You can optionally run `make html` from `docs/` to check the generate documentation related to the parser.